### PR TITLE
Fixtures

### PR DESCRIPTION
**Pull request by alexkiburu**
**WHAT**
 - add json props to config.json file
 - change the mysql connection string to get details from the config file
 - delete printlns
 - delete unused api methods
 - rename JWTAuthorize to TokenAuthorize
 - add a .gitignore file

**WHY**
To be able to read props from a config file.
To be able to use the mysql props dynamically by use of a config file

### CODE REVIEW

File: conf/props.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    	"DbPassword": "restaurant",
+        "DbName" : "restaurant",
+    	"DbHost": "127.0.0.1",
+    	"DbPort": "5432"
+    },
+    "InfoLogLevel" : "INFO",
+    "ErrorLogLevel" : "ERROR",
+    "InfoLogFile" : "path/to/info.log",
+    "ErrorLogFile" : "path/to/error.log"
+}
```
Comment (maretekent) [inline conf/props.json:18]: Good job

File: src/com/glosoftgroup/mpesa/api/Endpoint.java [hunk 2]
```diff
@@ -14,7 +14,8 @@
      
     public Endpoint() {
         /* change the BASE_URL accordingly */
-        BASE_URL = "http://127.0.0.1:8000/";
+//        BASE_URL = "http://127.0.0.1:8000/";
```
Comment (maretekent) [inline src/com/glosoftgroup/mpesa/api/Endpoint.java]: remove the commented code

File: src/com/glosoftgroup/mpesa/api/Api.java [hunk 3]
```diff
@@ -290,8 +234,6 @@ public JSONArray getNewTransactions(){
             URL url = new URL(Endpoint.getInstance().getNewMPESATransactionsURL());
             String response = sendGet(url);
             JSONParser parser = new JSONParser();
-//            JSONObject objectData = (JSONObject) parser.parse(response);
-//            parseData = (JSONArray) objectData.get("results");
             parseData = (JSONArray) parser.parse(response);
         } catch (Exception ex) {
             log.error(ex.getMessage(), ex);
```
Comment (maretekent) [inline src/com/glosoftgroup/mpesa/api/Api.java:239]: well done on logging errors

File: src/com/glosoftgroup/mpesa/utils/Props.java [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+@SuppressWarnings({"FinalClass", "ClassWithoutLogger"})
+public final class Props {
+    /**
+     * The properties file.
+     */
+    private static final String PROPS_FILE = "conf/props.json";
```
Comment (maretekent) [inline src/com/glosoftgroup/mpesa/utils/Props.java:22]: nice

**Review by maretekent — CHANGES_REQUESTED** [review]
Great job
A few things I would like you to rectify
- Lets stop pushing compiled pieces of code to git e.g class, jar files
- check on other inline comments

**Review by maretekent — COMMENTED** [review]
use .gitignore to bypass anything in build folder and all compiled classes
If already submitted git rm -r files check docs
```git rm --cached file1.txt
git commit -m "remove file1.txt"```

File: src/com/glosoftgroup/mpesa/api/Endpoint.java (hunk 2 above)
Comment (alexkiburu) [reply src/com/glosoftgroup/mpesa/api/Endpoint.java]: will do so.

File: src/com/glosoftgroup/mpesa/utils/Props.java (hunk 4 above)
Comment (alexkiburu) [reply src/com/glosoftgroup/mpesa/utils/Props.java:22]: Thank you

**Comment by alexkiburu** [comment]
@maretekent fixed the changes

**Review by maretekent — APPROVED** [review]
Look ok

**Pull request merged**